src/AzureAuth/CommandMain.cs

```diff
@@ -394,35 +396,36 @@ private int GetToken()
 
                     try
                     {
-                        result = authFlow.GetTokenAsync().Result;
+                        results = authFlowExecutor.GetTokenAsync().Result;
+                        succeededResult = results.FirstOrDefault(result => result.Success == true);

```

Comment:
I can't quite put my finger on why, but `succeededResult` sounds a little awkward to me while `successfulResult` sounds fine. 😆

Reply:
I will update it.